src/VisualStudio/VisualStudioInteractiveComponents/VisualStudioInteractiveComponents.csproj

```diff
@@ -58,6 +58,10 @@
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
     </ProjectReference>
+    <ProjectReference Include="..\..\Interactive\EditorFeatures\VisualBasic\BasicInteractiveEditorFeatures.vbproj"> 
+        <Project>{849e516a-595f-474b-adb4-e099f921cedf}</Project>
+        <Name>BasicInteractiveEditorFeatures</Name>
```

Comment:
It was being packaged at some point, and failure to do so when I put this project into a sub directory.